src/forms/CurrentExpenses.js

```diff
@@ -109,43 +109,46 @@ const EarnedFrom = function ({ hasExpenses, CashFlowRow, label, propData }) {
 
 const Utilities = function ({ current, type, time, updateClientValue }) {
 
-  let climate     = current.climateControl,
-      electricity = current.nonHeatElectricity,
-      phone       = current.phone,
-      fuelAssist  = current.fuelAssistance;
+  let hasClimate     = current.climateControl,
+      hasElectricity = current.nonHeatElectricity,
+      hasPhone       = current.phone,
+      hasFuelAssist  = current.fuelAssistance;
 
   let setChecked = function (evnt, inputProps) {
+    // Why is `checked` not always true here? Not a radio?
```

Comment:
Makes sense to me - you'd want to know if a checkbox got unchecked, right? But I guess a radio button only sends an event if it's the one that got selected (is this true?)

Reply:
I think it's true, but I think in semantic-ui-react `Radio` button are just a `Checkbox` under the hood. I understand how it's trying to mimic HTML, but I always found that annoying in HTML too...